src/main/java/me/zeroeightsix/fiber/constraint/ValuedConstraint.java

```diff
@@ -1,5 +1,11 @@
 package me.zeroeightsix.fiber.constraint;
 
+/**
```

Comment:
Not filled in?

Reply:
Looks like I missed this one. Nice catch.